extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -113,6 +118,18 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
   }
 
   /**
+   * @param {string} format
+   * @return {string} The ad unit hash key string.
+   * @private
+   */
+  adKey_(format) {
```

Comment:
This is not the DOM fingerprint. The `adk` parameter is used by AdSense and DoubleClick, and is thought of as the "ad block key." It is slightly different from `adf`. Both AdSense and DoubleClick use both parameters, but `adf `is the same for both, while `adk` is different between AdSense and DoubleClick. Since `adk` includes `adf` information, I put them both in this PR. I will add mention of `adk` to the PR summary.